src/app/core/database/mock-database.ts

```diff
@@ -37,6 +31,7 @@ export class MockDatabase extends Database {
     return instance;
   }
 
+  private views: { [key: string]: { map: string; reduce?: string } } = {};
```

Comment:
That type confuses me. Can you maybe elaborate (comment)/make the type more clear?